runelite-client/src/main/java/net/runelite/client/plugins/microbot/woodcutting/AutoWoodcuttingScript.java

```diff
@@ -74,7 +76,7 @@ public boolean run(AutoWoodcuttingConfig config) {
                     return;
                 }
 
-                if (Rs2Player.isMoving() || Rs2Player.isAnimating() || Microbot.pauseAllScripts || Rs2AntibanSettings.actionCooldownActive)
```

Comment:
Is there a reason why this has been removed here?

Reply:
Duplicate check, if you look there is an if statement right after this one for that condition